configuration/external_parameters.rst

```diff
@@ -13,119 +13,114 @@ you to easily do this.
+variable in you service container configuration, you can reference it using
+``%env(DATABASE_HOST)%`` in your configuration files:
 
 .. configuration-block::
 
     .. code-block:: yaml
 
+        # app/config/config.yml
         doctrine:
             dbal:
-                driver:   pdo_mysql
-                dbname:   symfony_project
-                user:     '%database.user%'
-                password: '%database.password%'
+                host: '%env(DATABASE_HOST)%'
 
     .. code-block:: xml
 
+        <!-- app/config/config.xml -->
         <!-- xmlns:doctrine="http://symfony.com/schema/dic/doctrine" -->
         <!-- xsi:schemaLocation="http://symfony.com/schema/dic/doctrine http://symfony.com/schema/dic/doctrine/doctrine-1.0.xsd"> -->
 
         <doctrine:config>
             <doctrine:dbal
-                driver="pdo_mysql"
-                dbname="symfony_project"
-                user="%database.user%"
-                password="%database.password%"
+                host="%env(DATABASE_HOST)%"
             />
         </doctrine:config>
 
     .. code-block:: php
 
+        // app/config/config.php
         $container->loadFromExtension('doctrine', array(
             'dbal' => array(
-                'driver'   => 'pdo_mysql',
-                'dbname'   => 'symfony_project',
-                'user'     => '%database.user%',
-                'password' => '%database.password%',
+                'host' => '%env(DATABASE_HOST)%',
             )
         ));
 
+You can also give the ``env()`` parameters a default value: the default value
+will be used whenever the corresponding environment variable is *not* found:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/parameters.yml
+        parameters:
+            database_host: '%env(DATABASE_HOST)%'
+            env(DATABASE_HOST): localhost
+
+    .. code-block:: xml
+
+        <!-- app/config/parameters.xml -->
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <container xmlns="http://symfony.com/schema/dic/services"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+            <parameters>
+                <parameter key="database_host">%env(DATABASE_HOST)%</parameter>
+                <parameter key="env(DATABASE_HOST)">localhost</parameter>
+            </parameters>
+         </container>
+
+    .. code-block:: php
+
+        // app/config/parameters.php
+        $container->setParameter('database_host', '%env(DATABASE_HOST)%');
+        $container->setParameter('env(DATABASE_HOST)', 'localhost');
+
+Setting environment variables is generally done at the web server level or in the
+terminal. If you're using Apache, Nginx or just the console, you can use e.g. one
+of the following:
+
+.. configuration-block::
+
+    .. code-block:: apache
```

Comment:
@wouterj thanks for the heads up. It will be merged in symfony.com soon:

![apache-sphinx](https://cloud.githubusercontent.com/assets/73419/19635979/8c8b194a-99c6-11e6-833f-2833aad90af0.png)